src/editor/components/components/ActionBar/ActionBar.component.jsx

```diff
@@ -0,0 +1,34 @@
+import { Circle20Icon, ArrowDown24Icon } from '../../../icons';
+import Events from '../../../lib/Events';
+
+import styles from './ActionBar.module.scss';
+
+const ActionBar = ({ handleAddClick, isAddLayerPanelOpen }) => {
+  const handleHandClick = () => {
+    Events.emit('hidecursor');
+  };
+
+  const handleSelectClick = () => {
+    Events.emit('showcursor');
+  };
+
+  return (
+    <div>
+      {!isAddLayerPanelOpen && (
+        <div className={styles.wrapper} id="findme">
+          <button type="button" onClick={handleSelectClick} key="handButton">
+            <ArrowDown24Icon />
+          </button>
+          <button type="button" onClick={handleHandClick} key="handButton">
```

Comment:
No need for key here. You only need to set key when you're iterating over an array.
And here you used the same key as the button just above. When you develop, always open the Chrome console to see the react warnings.